modules/deeptools_bamFingerprint.cfmod.pl

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env perl
```

Comment:
Is it worth rolling the functionality of this module into `modules/deeptools_bamCoverage.cfmod.pl`? It seems like much of the code is the same, so could keep things simpler. Is there a use case where people would want to run one and then the other (can this run on output from different sources, for example?)